app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+# == Schema Information
+#
+# Table name: users
+#
+#  id         :integer          not null, primary key
+#  uid        :string           not null
```

Comment:
noting here for future reference: I was wrong and `uid` is _not_ the kerbid. As such, we are leaving this as `User.uid`.